sdk/playwrighttesting/create-microsoft-playwright-testing/test/initialize.spec.ts

```diff
@@ -0,0 +1,195 @@
+import chai from "chai";
```

Comment:
suggestion (feel free to ignore): `@azure-tools/test-utils` exports a version of chai that already configures `chai-as-promised` and `chai-exclude` 

You'd just want to `import { expect } from "@azure-tools/test-utils"` and you can skip the chai.use call etc.

Reply:
moved from `chai` and `chai-as-promised` to `@azure-tools/test-utils`